src/Random/LCG.purs

```diff
@@ -72,11 +72,13 @@ lcgM :: Int
 lcgM = 2147483647
 
 -- | Perturb a seed value
-lcgPerturb :: Number -> Seed -> Seed
+-- | Note that `Int` operations are truncated to 32-bits, so we convert to
+-- | `Number` for this calculation to avoid overflow errors.
```

Comment:
```suggestion
-- Note that `Int` operations are truncated to 32-bits, so we convert to
-- `Number` for this calculation to avoid overflow errors.
```